front50-api/src/main/java/com/netflix/spinnaker/front50/api/model/pipeline/Trigger.java

```diff
@@ -28,6 +32,76 @@ protected Map<String, Object> delegate() {
     return backing;
   }
 
+  public Boolean getEnabled() {
+    Object enabled = backing.get("enabled");
+    if (enabled instanceof Boolean) {
+      return (Boolean) enabled;
+    }
+
+    // orca's DependentPipelineExecutionListener.groovy currently (18-apr-23)
```

Comment:
See [here](https://github.com/spinnaker/orca/blob/da3cf93de4f003a91d3ddd01c1996d0a8961e58f/orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/spring/DependentPipelineExecutionListener.groovy#L97) for the orca code.